hack/assets/fonts/fira-code/webfont.css

```diff
@@ -0,0 +1,45 @@
+/* Source: https://github.com/tonsky/FiraCode */
+
+@font-face{
+    font-family: 'Fira Code';
```

Comment:
Do you think that we still need to support all of these web font file types?  Based upon https://github.com/source-foundry/Hack/issues/286#issuecomment-325029837 and https://github.com/source-foundry/Hack/issues/286#issuecomment-325029926 it seems that we can simplify this to woff and woff2 since there is support in nearly all modern browsers at this stage.  Thoughts?  

Not necessary for this PR but a thought for down the road.

Reply:
Let’s remove TTF, EOT and SVG (if any) at a later stage and open an issue for it.

Reply:
agree!